zulip/integrations/rss/rss-bot

```diff
@@ -248,7 +251,7 @@ for feed_url in feed_urls:
             # entries in reverse chronological order.
             break
 
-        feed_name = opts.topic
+        feed_name = opts.topic or data.feed.title or feed_url  # type: str
```

Comment:
I would expect `feed_url` to often be too long; do we want to clamp too long topics here in the client code?

Reply:
not sure what your suggestion is here really.

`data.feed.title or feed_url` was already here before my PR